addons/binding/org.openhab.binding.mihome/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mihome"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Xiaomi Mi Smart Home Binding</name>
+    <description>Connects to Xiaomi Smart Gateway v2 and allows to communicate with Xiaomi Mi Smart Home suite (also called Xiaomi Aqara). 
+    The Gateway connects with a range of Xiaomi Zigbee sensors such as Wireless Switch, Wall Plug or Temperature Sensor.</description>
+    <author>Patrick Boos</author>
```

Comment:
The field doesn't allow to include more people so we can either remove it altogether or keep it as it is.
I don't mind leaving it. This PR is already half year old 😄